styles.css

```diff
@@ -0,0 +1,186 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* Media screen for mobiles with minimum width of 375px */
+@media screen and (min-width: 375px) {
+  body {
+    background-image: url(./images/header-shapes-mobile.png);
+    background-size: contain;
+    font-family: 'Poppins', sans-serif;
+    background-repeat: no-repeat;
+  }
+
+  header {
+    padding: 20px;
+    display: flex;
+    align-items: center;
+  }
+
+  .logo-text {
+    width: 76px;
+    height: 20px;
+    color: #6070ff;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 700;
+  }
+
+  nav {
+    display: flex;
+    flex: 1;
+    text-align: right;
+    font-style: normal;
+  }
+
+  nav ul {
+    display: none;
+    list-style-type: none;
+  }
+
+  nav ul li {
+    display: inline-block;
+    margin-right: 20px;
+  }
+
+  a {
+    text-decoration: none;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .bigtext1 {
+    display: none;
+  }
+
+  .bigtext2 {
+    color: #172b4d;
+    font-size: 40px;
+    width: 327px;
+    height: 150px;
+    font-weight: bolder;
+    font-style: normal;
+  }
+
+  .small-text {
+    color: #344563;
+    font-size: 16px;
+    width: 327px;
+    height: 146px;
+    margin-top: 40px;
+    padding-right: 10px;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  #introduction {
+    padding-left: 5%;
+    padding-top: 10%;
+  }
+
```

Comment:
- [ ] Please make sure that there is no margin between the `#introduction` section and the view screen in the mobile section. I suggest using width: 100% on your section as mentioned in the [requirement]()  
>  Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

<img width="372" alt="Screenshot 2023-02-16 at 10 08 52 AM" src="https://user-images.githubusercontent.com/44798044/219276844-69685f91-b9ea-4250-8cea-8d484b8a2a44.png">